### PR TITLE
Clean up scenario configs

### PR DESCRIPTION
**Pull request by FelicitasBeier**
## :bird: Description of this PR :bird:

- We split off the project-specific scenario settings from the main scenario_config.csv into single scenario_config_project.csvs to clean up rows and columns in the csv files and avoid merge conflicts in the configs in the future

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [x] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

**Comment by FelicitasBeier** [comment]
Please wait with merging this PR until all people responsible for the projects have reviewed their config files and updated their start scripts if necessary.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,16 +8,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### changed
```
Comment (tscheypidi) [inline CHANGELOG.md:9]: please add an entry for the config handling change

**Review by tscheypidi — APPROVED** [review]
looks good. I am only missing an entry in the CHANGELOG.md

File: CHANGELOG.md (hunk 1 above)
Comment (pvjeetze) [reply CHANGELOG.md:9]: +1

**Review by pvjeetze — APPROVED** [review]
Looks fine for me

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,16 +8,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+- **scripts** rewrite of merge_report.R based on rds files and rbind, which allows for more flexibility when merging reports. Avoid inconsistent use of "GLO" instead of "World" in report.rds files.
 - **15_food** revision of MP/SCP implementation for milk and meat alternatives. Added demand for fat and sugar as ingredients for MP-based milk alternatives. Added optional demand for fat as ingredient for MP-based meat alternatives.
-- **script** scripts/start_functions.R decide individually for demand and price whether they are read from a REMIND report.
+- **scripts** scripts/start_functions.R decide individually for demand and price whether they are read from a REMIND report.
 - **80_optimization** abort GAMS in case of execution errors
-- **script** updated EATLancet project start scripts
-- **script** replaced gdx package with gdx2 package calls
+- **scripts** updated EATLancet project start scripts
+- **scripts** replaced gdx package with gdx2 package calls
+- **scripts** split scenario_config into project-specific configs
```
Comment (k4rst3ns) [inline CHANGELOG.md:17]: We added this line. What is missing for you in the CHANGELOG? @tscheypidi

Comment (tscheypidi) [reply CHANGELOG.md:17]: I probably overlooked it.

Comment (pvjeetze) [reply CHANGELOG.md:17]: Maybe change line 17 to **config** instead of **scripts**.

Comment (pvjeetze) [inline CHANGELOG.md:17]:
```suggestion
- **config** split scenario_config into project-specific configs
```

**Pull request merged**